src/Explorer/Panes/SettingsPane/SettingsPane.tsx

```diff
@@ -185,6 +191,25 @@ export const SettingsPane: FunctionComponent = () => {
             </div>
           </div>
         )}
+        <div className="settingsSection">
```

Comment:
The checkbox styling in the settings panel looks off:
![image](https://user-images.githubusercontent.com/56978073/228934054-8470a24c-bd12-489a-bb55-39345ccbfd13.png)

1. The checkbox should be in the same line as the label:
![image](https://user-images.githubusercontent.com/56978073/228934171-75af2a9c-cacd-4584-b5b0-bff5cbb96b22.png)
2. The tooltip info icon is not aligned properly with the label text.